.rat-excludes

```diff
@@ -35,6 +35,7 @@ dagre-d3.min.js
 graphlib-dot.min.js
 sorttable.js
 vis.min.js
+vis.js
```

Comment:
It's because `vis.js` is referred by `vis.map` and `vis.map` is referred by vis.min.js.
`vis.js` is only downloaded when Source Maps is enabled for debug.
Firefox or Chrome is supported SourceMaps and browsers which don't support Source Maps just ignore the reference directive in vis.min.js.